cnvlib/gary.py

```diff
@@ -204,6 +205,11 @@ def autosomes(self, also=()):
 
     def by_chromosome(self):
         """Iterate over bins grouped by chromosome name."""
+
+        # necessary because of pandas bug which says:
```

Comment:
I think these two below are tightly related - but I had no time yet to hunt down the bug in detail and confirm how they are related.

* pydata/pandas#10505
* pydata/pandas#10508